app/Meeting.php

```diff
@@ -102,6 +102,10 @@ public function startMeeting()
             $meetingParams->addPresentation($file->getDownloadLink(), null, preg_replace("/[^A-Za-z0-9.-_\(\)]/", '', $file->filename));
         }
 
+        if (empty($meetingParams->getPresentations()) && !empty(setting('default_presentation'))) {
+            $meetingParams->addPresentation(setting('default_presentation'));
```

Comment:
I would prefer a presentation name, so BBB does not use this random name. We could hardcode "default.pdf", use a translated string or a custom variable. @dennis95stumm What do you think?

Reply:
I've hardcoded the name with the extension from the client!

Reply:
That's a great solution, I did not even think about the extension